pages/common/nkf.md

```diff
@@ -0,0 +1,27 @@
+# nkf
+
+> Convert character code and new line code.
+
+- Stdout utf-8 code:
+
+`nkf -w {{path/to/file.txt}}`
+
+- Stdout shift_jis code:
```

Comment:
same as above.

Reply:
Seems to be another encoding from https://linux.die.net/man/1/nkf. I would suggest the same format as above.

Reply:
Done.